modules/apps/remote-web-app/remote-web-app-service/service.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.3.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_3_0.dtd">
+
+<service-builder auto-import-default-references="false" auto-namespace-tables="false" dependency-injector="ds" mvcc-enabled="true" package-path="com.liferay.remote.web.app">
+	<namespace>RemoteWebApp</namespace>
+	<entity local-service="true" name="RemoteWebAppEntry" remote-service="false" trash-enabled="false" uuid="true">
+
+		<!-- PK fields -->
+
+		<column container-model="true" name="entryId" primary="true" type="long" />
+
+		<!-- Group instance -->
+
+		<!--column name="groupId" type="long" /-->
+
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+		<column name="userId" type="long" />
+		<column name="userName" type="String" uad-anonymize-field-name="fullName" />
+		<column name="createDate" type="Date" />
+		<column name="modifiedDate" type="Date" />
+
+		<!-- Other fields -->
+
+		<column name="name" type="String" uad-nonanonymizable="true" />
```

Comment:
For now, name and url is enough to demonstrate the functionality. Then we can add more fields as necessary.